packages/runtime-vapor/__tests__/dom/templateRef.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { createComponent, ref, setRef, template } from '../../src'
+import { makeRender } from '../_utils'
+
+const define = makeRender()
+
+describe('api: template ref', () => {
+  test('ref on element', () => {
```

Comment:
We need to make sure the tests are consistent with runtime-core
https://github.com/vuejs/core/blob/958286e3f050dc707ad1af293e91bfb190bdb191/packages/runtime-core/__tests__/rendererTemplateRef.spec.ts#L16